website/docs/operator/data-recovery.md

```diff
@@ -0,0 +1,7 @@
+---
+id: data-recovery
+title: Data Recovery
+description: Provides documentation for Operators to recover missing data.
+---
+
+This document in under construction.

```

Comment:
Did you want to add information on Data Recovery? Maybe something from https://wiki.earthdata.nasa.gov/display/PCESA/Disaster+Recovery+Operator+Guide or here https://wiki.earthdata.nasa.gov/display/LPCUMULUS/LPCumulus+Notes+on+Disaster+Recovery

Reply:
Will leave stubbed out, put in a card for content on this page, and work with Operators to develop content.